libs/island-ui/core/src/lib/Tag/Tag.tsx

```diff
@@ -14,6 +14,7 @@ export type TagVariant =
   | 'rose'
   | 'blueberry'
   | 'dark'
+  | 'mint'
```

Comment:
We used to have this mint variant on tags but we removed it because the color contrast was not accessible (see here https://webaim.org/resources/contrastchecker/?fcolor=00B39E&bcolor=FFFFFF (this is the darkest variation of mint and it still fails.))

Reply:
Ah i see - fixed

Reply:
This can work if the font color is black and the background is lighter mint ... let's see what the design team thinks of that combination.

Reply:
Removed for now